kpi/models/import_export_task.py

```diff
@@ -600,9 +601,8 @@ def _run_task(self, messages):
     def _filter_by_source_kludge(queryset, source):
         """
         A disposable way to filter a queryset by source URL.
-        TODO: make `data` a `JSONBField` and use proper filtering
         """
-        return queryset.filter(data__contains=source)
+        return queryset.filter(data__source=source)
```

Comment:
great, but it seems like `_filter_by_source_kludge()` is used only in `kpi/tests/test_mock_data_exports.py`. maybe we trash this method and use `filter(data__source=source)` directly in the tests? (or at least remove `_kludge` from the method name :smile:)

Reply:
indeed